CMakeLists.txt

```diff
@@ -26,7 +26,19 @@ set(version 1.1.0)
 project(composable_kernel VERSION ${version} LANGUAGES CXX HIP)
 include(CTest)
 
-find_package(Python3 3.6 COMPONENTS Interpreter REQUIRED)
+if(NOT CK_USE_ALTERNATIVE_PYTHON)
+   find_package(Python3 3.6 COMPONENTS Interpreter REQUIRED)
```

Comment:
@illsilin should here be 3.8? Because no alternative is required then we need 3.8 to work

Reply:
No, if we require that the minimum version of Python is 3.8, the builds will fail at this point. 
The way this works is: we allow the cmake to locate the older python packages that are installed in the default paths, but then swap the executable with the newer version that is installed elsewhere (and also add the necessary runtime library to the LD_LIBRARY_PATH to make it work).